converter/lookml/measure.py

```diff
@@ -0,0 +1,61 @@
+from typing import Dict, List, Optional
+from dbt_semantic_interfaces.implementations.elements.measure import PydanticMeasure
+from dbt_semantic_interfaces.type_enums.aggregation_type import AggregationType
+from pydantic import BaseModel
+
```

Comment:
I know this repo has a lot of dangling functions, but it might be helpful to encapsulate them in classes! e.g., in this file the functions could be made `staticmethods` on the `LkmlMeasure` class

Reply:
Interesting thought! That seems like a minor stylistic change. In this case, I think I prefer the functions as standalone because they don't input our return a `LkmlMeasure`. I'm always happy to refactor for organization in the future, though.